detection_rules/docs.py

```diff
@@ -249,6 +249,10 @@ def code(cls, value: str, code='js'):
         line_sep = "-" * 34
         return f'[source, {code}]\n{line_sep}\n{value}\n{line_sep}'
 
+    @classmethod
+    def content(cls, value: str):
```

Comment:
```suggestion
    def content(cls, value: str) - > str:
```
Should we add type hint?

Reply:
Removed this class method.